app/models/question.rb

```diff
@@ -10,5 +10,15 @@ def initialize(question_hash)
   def answers_for_collection
     answers.map { |answer| answer['text'] }
   end
+
+  def redirect_path_for_answer(chosen_answer)
+    answer_hash = answers.detect { |answer| answer['text'] == chosen_answer }
+
+    if answer_hash.key?('goto')
+      Rails.application.routes.url_helpers.questions_path(answer_hash['goto'])
```

Comment:
I mainly chose it because it has the same number of characters as `text` - so it's possibly easier to read when you're editing the file as a non-developer.

Reply:
As discussed, I've changed `goto` to `next`.